### PR TITLE
Change Text to Label and cast all sizes to int

### PR DESCRIPTION
**Pull request by Appesteijn**
Together with Uranium PR: https://github.com/Ultimaker/Uranium/pull/293

CURA-4437

### CODE REVIEW

File: resources/qml/Preferences/MaterialView.qml [hunk 1]
```diff
@@ -108,15 +108,15 @@ TabView
                 {
                     width: scrollView.columnWidth;
                     height:  parent.rowHeight;
-                    spacing: UM.Theme.getSize("default_margin").width/2
+                    spacing: parseInt(UM.Theme.getSize("default_margin").width/2)
```
Comment (sedwards2009) [inline resources/qml/Preferences/MaterialView.qml]: `Math.floor()`

Comment (fieldOfView) [reply resources/qml/Preferences/MaterialView.qml]: I agree that Math.floor() is better. I think parseInt will first cast the value to a string, and then parse it back to a number again. I used | 0 as shorthand for Math.floor() elsewhere, but that's a little obscure.

Comment (ChrisTerBeke) [reply resources/qml/Preferences/MaterialView.qml]: We found the | 0's indeed, but they're easily mistaken for || 0 (yay JS syntax). We can use Math.floor indeed.

Comment (Appesteijn) [reply resources/qml/Preferences/MaterialView.qml]: I ll replace all parseInt() with Math.floor()

**Pull request merged**